src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -145,11 +155,15 @@ const ChatItemMyTextV2 = ({
 
   const renderAvatar = React.useCallback(() => {
     if (isContinuous) return <View style={styles.avatar} />;
-    if (AnonymousImage) return AnonymousImage;
+    if (AnonymousImage) return <View style={styles.ml8}>{AnonymousImage}</View>;
     return <FastImage style={styles.avatar} source={{uri: avatar}} />;
   }, []);
 
-  const textContainerStyle = [styles.textContainer, isNewLine && styles.textContainerNewLine];
+  const textContainerStyle = [
+    chatType === SIGNED ? styles.containerSigned : styles.containerAnon,
+    styles.textContainer,
+    isNewLine && styles.textContainerNewLine
+  ];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for rendering the avatar has been updated. The `AnonymousImage` is now wrapped in a `View` with style `ml8`. Also, the `textContainerStyle` now depends on the `chatType`. Ensure that these changes don't break the layout of the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->